### ISSUE TITLE
Reshape error: 1 -> 115200

### ISSUE THREAD

**Issue by ldol31627** [alibaba/MNN#1493]
ONNX 模型转换出错，请帮忙看下是什么原因？opset=11
mnnconvert -f ONNX --modelFile lane.onnx --MNNModel lane.mnn
[08:24:07] :31: ONNX Model ir version: 6
Start to Optimize the MNN Net...
Reshape error: 1 -> 115200
Reshape error: 1 -> 115200
[08:24:07] :64: Inputs: mask
[08:24:07] :64: Inputs: image
[08:24:07] :69: Outputs: 815
[08:24:07] :69: Outputs: 1507
[08:24:07] :69: Outputs: pred_curves
[08:24:07] :69: Outputs: 1509
[08:24:07] :69: Outputs: pred_logits

这是模型文件：
[lane.zip](https://github.com/alibaba/MNN/files/6535660/lane.zip)

**Comment by jxt1234** [comment]
应该是已经修复的问题，用最新的 C++ 代码编译模型转换工具转换吧，python 版本还没有更新

**Comment by jxt1234** [comment]
最新代码测试了一下，没有问题
